content/introduction/how-ipfs-works.md

```diff
@@ -0,0 +1,82 @@
+---
+title: How IPFS Works
+weight: 2
+---
+
+IPFS is a peer-to-peer (p2p) network that links content-addressed data to other content-addressed data. Content is accessible through peers that might relay information or store it (or do both!), and those peers can be located anywhere in the world. IPFS knows how to find what you ask for by its content address, rather than where it is.
+
+<!-- <img alt="IPFS Stack" src="../assets/ipfs_stack.png" width="200px" /> -->
+
+## There are three important things to understand about IPFS
+
+Let’s first dive in to _content addressing_ and how that content is _linked together_. This “middle” part of the IPFS stack is what connects the ecosystem together; everything is built on being able to find content via linked, unique identifiers.
+
+### 1 \\ Content addressing and linked data
+
+<!-- <img alt="IPFS Stack - Data Structures" src="../assets/ipfs_stack-data.png" width="200px" /> -->
+
+IPFS uses _content addressing_ to identify content by what’s in it, rather than by where it’s located. Looking for an item by content is actually something you do all the time. For example, when you look for a book in the library, you ask for it by the title; that’s content addressing because you’re asking for **what** it is. If you were using location addressing to find that book, you’d ask for it by **where** it is: “I want the book that’s on the second floor, first stack, third shelf from the bottom, four books from the left.” If someone moved that book, you’d be out of luck!
```

Comment:
```suggestion
IPFS uses _content addressing_ to identify content by what’s in it, rather than by where it’s located. Looking for an item by content is actually something you do all the time. For example, when you look for a book in the library, you ask for it by the title/author/genre/story; that’s content addressing because you’re asking for **what** it is. If you were using location addressing to find that book, you’d ask for it by **where** it is: “I want the book that’s on the second floor, first stack, third shelf from the bottom, four books from the left.” If someone moved that book, you’d be out of luck!
```

Reply:
@momack2 I think for the sake of clarity the example is better with just `title` listed.

Reply:
I left `title` to keep it simple, but added an `often` to give it some unnamed other ways to ask for a book.